### ISSUE TITLE
Question regarding the reported performance

### ISSUE THREAD

**Issue by dain5832** [HKUDS/DCCF#1]
Hi,
I was impressed in your previous work, LightGCL and DCCF and noticed these papers are from the same lab!
Thanks for sharing interesting works.
While reading the paper, I found that the performance you have reported in the DCCF paper is quite different from the one you have reported in LightGCL.

In LightGCL, the performance for gowalla is reported as 0.15 and the code you have uploaded achieves around 0.21.
However, in DCCF, the performance of LightGCL is reported as 0.18.
Could you give me an explanation regarding where these differences came from?

Thanks!

**Comment by Re-bin** [comment]
Hi, Sure!

Thanks for your interests in our work. To ensure a fair comparison, we set the embedding size to 32 for all baselines. However, in the latest version of LightGCL's code, the embedding size was set to 64. Additionally, we trained LightGCL using the same training process as DCCF, with a uniform batch size of 10240. It also affects the performance of LightGCL, which differs from that of DCCF. Furthermore, LightGCL has undergone several performance updates, including the removal of residual links, since its initial release. Thus, the results we report are based on the original paper's implementation, which may result in performance differences.

I hope the above answers can help with your confusion.

Best,
Xubin
